File: tests/php/example2.php
<?php

// Fonction pour transformer le texte
function TransformText($text) {
    $text = strtoupper($text); // Convertir en majuscules
    $prefix = 'Prefix: ';
    return AddPrefix($modifiedText, $prefix);
}

// Fonction pour ajouter un préfixe
function AddPrefix($text, $prefix) {
    return $prefix . $text;
}

// Fonction de test
function test() {
    $inputText = 'Hello, World!';
    $result = TransformText($inputText);
    echo $result . '\n';
}

// Fonction principale
test();

?>

File: tests/rb/example2.rb
# Fonction pour transformer le texte
def TransformText(text)
  text = text.upcase # Convertir en majuscules
  prefix = "Prefix: "
  AddPrefix(modified_text, prefix)
end

# Fonction pour ajouter un préfixe
def AddPrefix(text, prefix)
  prefix + text
end

# Fonction de test
def test
  input_text = "Hello, World!"
  result = TransformText(input_text)
  puts result
end

# Fonction principale
test
